touchscreen_electron/pages/css/voting.css

```diff
@@ -1,8 +1,9 @@
 #load-questions-button,
-#start-BLE-button,
+#step-II-2,
 #voting-token-button,
 #voting-token-input,
 #step-V,
-#step-II-1 {
+#step-II-1,
+#step-II-0 {
   visibility: hidden;
```

Comment:
I think amber switched all the visibility stuff to display instead maybe we should follow suit.